src/App.jsx

```diff
@@ -1,107 +1,32 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUser(id) {
+  return usersFromServer.find(
+    user => user.id === id,
```

Comment:
it's worth return `null` if no element found. Otherwise the function will return `undefined`

Reply:
Good tip, please fix